File: 151003/Matoshko/tweet-services/tweet-service-publisher/src/main/java/services/tweetservice/domain/entity/TweetSticker.java
package services.tweetservice.domain.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.FetchType;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@Entity
@Table(name = "tbl_tweet_sticker")
public class TweetSticker extends BaseEntity {
    @JoinColumn(name = "tweet_id")
    @ManyToOne(fetch = FetchType.LAZY)
    private Tweet tweet;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "sticker_id")
    private Sticker sticker;
}

File: 151004/Dubovski/Lab5/publisher/DC_Lab1/DC_Lab1/Models/TweetSticker.cs
using DC_Lab1.Models;
using System;
using System.Collections.Generic;

namespace DC_Lab1;

public partial class TweetSticker
{
    public int Id { get; set; }

    public int? tweetId { get; set; }

    public int? stickerId { get; set; }

    public virtual Sticker? Sticker { get; set; }

    public virtual Tweet? Tweet { get; set; }
}
